src/chan_vese.jl

```diff
@@ -0,0 +1,254 @@
+"""
+    chan_vese(img; [μ], [λ₁], [λ₂], [tol], [max_iter], [Δt], [reinitial_flag])
+
+Segments image `img` by evolving a level set. An active contour model 
+which can be used to segment objects without clearly defined boundaries.
```

Comment:
Maybe also worth documenting that this algorithm supports 3d images, and not yet don't support `RGB` images.